packages/ffe-file-upload-react/src/FileUpload.js

```diff
@@ -29,97 +33,106 @@ class FileUpload extends React.Component {
         this.props.onFilesSelected(event.target.files);
     }
 
+    onFilesDropped(event) {
+        event.preventDefault();
+        this.setState({ hover: false });
+        this.props.onFilesDropped(event);
+    }
+
     onFileDeleted(event) {
-        this.props.onFileDeleted(
-            this.props.selectedFiles.find(
-                file => file.name === event.target.id,
-            ),
-        );
+        this.props.onFileDeleted(this.props.files[event.target.id]);
     }
 
     render() {
         const {
             id,
             label,
-            selectedFilesHeaderLabel,
-            selectedFiles,
-            accept,
+            files,
+            cancelText,
+            deleteText,
             multiple,
-            errorMessage,
-            infoMessage,
-            successMessage,
+            infoTitle,
+            infoText,
+            infoSubText,
+            uploadTitle,
+            uploadMicroText,
+            uploadSubText,
+            onFileDeleted,
         } = this.props;
 
         return (
             <div className="ffe-file-upload">
-                <button
-                    className="ffe-file-upload__button"
-                    aria-invalid={String(!!errorMessage)}
-                    onClick={this.triggerUploadFileNativeHandler}
+                <div className="ffe-file-upload__info-section">
+                    <div className="ffe-file-upload__info-section__title">
+                        {infoTitle}
+                    </div>
+                    {Object.keys(files).length > 0 ? (
+                        <div className="ffe-file-upload__info-section__selected-files">
+                            {Object.keys(files).map((file, index) => (
+                                <FileItem
+                                    key={index}
+                                    index={index}
+                                    file={files[file]}
+                                    cancelText={cancelText}
+                                    deleteText={deleteText}
+                                    onFileDeleted={onFileDeleted}
+                                />
+                            ))}
+                        </div>
+                    ) : (
+                        <>
+                            <div className="ffe-file-upload__info-section__text">
+                                {infoText}
+                            </div>
+                            <div className="ffe-file-upload__info-section__subtext">
+                                {infoSubText}
+                            </div>
+                        </>
+                    )}
+                </div>
+                <div
+                    className="ffe-file-upload__upload-section"
+                    role="presentation"
```

Comment:
`role="presentation"` will remove the semantic meaning of the element and all child elements, I'm not sure that's needed here. How assistive tech handles it varies as far as I know but I don't think it helps in this context.